west.yml

```diff
@@ -63,7 +63,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: protocols_serialization_2
+      revision: pull/1894/head
```

Comment:
```suggestion
      revision: a56fee6870e152b71a7baf5b85bfa0b9746286a0
```

Reply:
But we need a single change from https://github.com/nrfconnect/sdk-zephyr/commit/d1f774f888cfdd8e20c389b6eab33953b6936766 (which btw. I'm going to post into Zephyr).

Reply:
yep, you are right